.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+/wondershaper.service
```

Comment:
Why is this added to the gitignore

Reply:
`*~` is Emacs backup files. They should arguably not go in this MR (Emacs users should just add `*~` (and `#*`) to their global ignores. Will add a fixup for this.

Reply:
`/wondershaper.service` is ignored since it is *generated* below (and hence we'd like to avoid committing it by accident).